homeassistant/components/ezviz/services.yaml

```diff
@@ -0,0 +1,49 @@
+ezviz_alarm_sound:
```

Comment:
Please update the service descriptions to our new format.

See: <https://developers.home-assistant.io/docs/dev_101_services/#service-descriptions>

Reply:
Done. I have updated the service descriptions to the new format.